book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md

```diff
@@ -49,6 +49,8 @@ The basic options are:
                        the flashed device The default is: quackquack
     --wifi             default: duckietown:quackquack
     --country          default: US
+    --type             The type of your device. E.g. `duckiebot`.
+    --configuration    The configuration of your robot. This is associated with `--type` option. E.g. `DB-beta`.
```

Comment:
for type and configuration maybe add all the (important) options?

Reply:
included watchtower, traffic_light and duckiebot for types. For configurations DB-beta, DB20, DB19, and DB18.